tiaas.md

```diff
@@ -8,43 +8,75 @@ title: Training Infrastructure as a Service
-training. When normal users run tools on our server, these jobs are instructed
-to avoid the training pools by default.
 
-When users join a training, using a special URL we provide you, they then are
-placed in a special training group. Their jobs will then preferentially run the
-jobs on a training machine, and, in the event there is no more capacity, they
-will run on the main queue. If a spot on a training VM opens up first, they
-will run there rather than continuing to wait in the main queue.
+## How TIaaS Works
 
-Before users can join a specific piece of training, they need to be logged into the [European Galaxy server](https://usegalaxy.eu/). Once the training is over, the data will stay available for further use. However, we encourage you to clean up all the histories whenever this data will not be used anymore.
+We have several "pools" of virtual machines (VMs) attached to UseGalaxy.eu that run user jobs. For trainings that request TIaaS, we attach a new pool of VMs that is specially labelled for that training. When users join a training using a __special URL__ we provide you, they are assigned to a __special training group__. Their jobs will then __preferentially run on a training machine__, and, in the event there is no more capacity, they
+will run on the main queue. If a spot on a training VM opens up first, they
+will run there rather than continuing to wait in the main queue. The jobs run by the rest of users on our server are instructed to avoid the training pools.
 
 Some more general information about the TIaaS service:
 
-- [a Calendar](https://usegalaxy.eu/tiaas/calendar/) that shows when TIaaS trainings are booked
-- [some Statistics](https://usegalaxy.eu/tiaas/stats/) about the TIaaS events
+- [A calendar](https://usegalaxy.eu/tiaas/calendar/) that shows when TIaaS trainings are booked.
+- [Some statistics](https://usegalaxy.eu/tiaas/stats/) about the TIaaS events.
+
+### Before the training
+
+Before users can join a specific training, they need to be __logged into the [European Galaxy server](https://usegalaxy.eu/)__.
+
+To __import data__:
+
+- If you are using the [GTN material](training.galaxyproject.org/) for
+your workshop, then all the [training data is already mirrored into Galaxy](https://usegalaxy.eu/library/folders/Fa21272e5bd712216).
+
+- If you are using your own data, you can upload it to your account
+into a history. Later, you can [make it accessible](https://usegalaxy.eu/histories/sharing) to your trainees (do not
+forget to also share the datasets by clicking `Also make all objects within the History accessible`). Your trainees will then be able to import
+your history and start working with your data.
+
+
+
+### After the training
+
+Once the training is over, the data will stay available for further use. However, we encourage you to __clean up all the histories__ whenever this data will not be used anymore.
+
+To keep running this service __free for you we need your feedback__. You can:
```

Comment:
to keep running this free service for your users we need your feedback and support, it would be great if you can ...